app/views/packages/show.html.erb

```diff
@@ -1,7 +1,8 @@
+
 <div class="container col-sm-6 col-md-8 col-lg-10" >
   <div class="row">
     <div class="package-header-box col-6">
-      <div class="package-header" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.3)), url(https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/breakfast.jpg)">
+      <div class="package-header" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.3)), url(<%= @package.hotel.photo %>)">
```

Comment:
This would look weird for now, since photo in the seeds are for avatars in index. In the future I'd suggest to add a new column `pictures` for an array of photos for your show page